### PR TITLE
Extend podLabel configurations

### PR DESCRIPTION
**Pull request by bobertrublik**
This PR enables :

- configuring pod labels for all pg exporter pods generated from each db-instances instance
- setting separate labels between db-operator's controller and webhook deployments

### CODE REVIEW

**Comment by bobertrublik** [comment]
Hmm do you have a clue why the action failed? It looks like it ran in some sort of timeout.

**Comment by allanger** [comment]
I think this is the reason, but I'm not sure yet
```
reading data from mysql...
ERROR 1045 (28000): Plugin caching_sha2_password could not be loaded: Error loading shared library /usr/lib/mariadb/plugin/caching_sha2_password.so: No such file or directory
reading data from mysql...
ERROR 1045 (28000): Plugin caching_sha2_password could not be loaded: Error loading shared library /usr/lib/mariadb/plugin/caching_sha2_password.so: No such file or directory
reading data from mysql...
ERROR 1045 (28000): Plugin caching_sha2_password could not be loaded: Error loading shared library /usr/lib/mariadb/plugin/caching_sha2_password.so: No such file or directory
```

**Comment by allanger** [comment]
I've fixed it here: https://github.com/db-operator/charts/pull/55
You'll have to rebase after that PR is merged (currently waiting for a review)

**Comment by bobertrublik** [comment]
Awesome thank you!

**Review by allanger — APPROVED** [review]

**Pull request merged**
